wakuv2/waku.go

```diff
@@ -1652,14 +1680,24 @@ func (w *Waku) seedBootnodesForDiscV5() {
 
 	for {
 		select {
+		case <-w.dnsDiscAsyncRetrievedSignal:
+			if canQuery() {
```

Comment:
in case of lightclient mode since discv5 won't be enabled, wondering if this will cause an issue as we are trying restart discv5.